### ISSUE TITLE
_abort() anp.cpp:635

### ISSUE THREAD

**Issue by ab9003** [comex/frash#8]
I get the error _about() anp.cpp:635 when testing nearly any flash content. After clicking to start anything it works fine but after clicking again to interact with it the error comes up. It seems to happen in nearly every flash game I tested. I have the latest libflashplayer.so armeabi not a7a. I installed frash via iFile and all the files look alright. Is it possible still I have a bad libflashplayer? Is anybody else having this issue?

**Comment by kangaroo** [comment]
You're trying to go fullscreen.  Do you have some urls where this happens?

**Comment by ab9003** [comment]
Newgrounds xgenstudios armor games all .com urls. I haven't tried going full screen but I have tried pinch zooming while flash was active.

**Comment by feisley** [comment]
I noticed this as well on newgrounds. It seems to be something in the page that is doing it because if I grab the url to the swf file directly and navigate to that in Safari, then it seems to work fine.

**Comment by vcazan** [comment]
What I noticed is you you run a flash file then go to the URL bar and then go back to the flash it stops working

**Comment by comex** [comment]
Yes, this is window_impl_requestFullScreen.  A possible temporary solution (aside from actually implementing fullscreen, heh) would be making that method do nothing.

**Comment by feisley** [comment]
comex,
Perhaps it was just more broken apis but i tried that for one site and it seemed to abort() in other places.

**Comment by ab9003** [comment]
It seems to happen whenever I focus on a flash document. For example run Jesus run on the new grounds front page. It was running fine when I first loaded it but when I tapped it to interact the 635 error came up. I didn't pinch zoom this time or anything.

**Comment by kangaroo** [comment]
Just comment out the abort for now like comex suggested.
